code/src/PostThoughts.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import './PostThoughts.css';
+
+export const PostThoughts = () => {
+  const THOUGHTS_URL = 'https://happy-thoughts-technigo.herokuapp.com/thoughts';
+  const [message, setMessage] = useState('');
+
+  const handleSubmit = event => {
+    event.preventDefault()
+
+    fetch(THOUGHTS_URL,
+      {
+        method: 'POST',
+        headers: { 'content-type': 'application/json' },
+        body: JSON.stringify({ message })
+      }
+    ).then(() => {
+      setMessage('')
+      window.location.reload()
+      })
+    }    
+
+    return (
+      <form onSubmit={handleSubmit} className="post-contents">
+        <h1
+          className='happyheader-text'
+          tabIndex='0'
+        >
+          What is making you happy right now?
+        </h1>
+        <textarea
+          placeholder='Enter stoke here!'
+          value={message}
+          tabIndex='0'
+          aria-label='Input for text'
+          className='form-input'
+          maxLength='140'
+          onChange={event => setMessage(event.target.value)}
+          required
+        >
+        </textarea>
+        <p className={((message.length < 5 || message.length > 140) ? "text-invalid" : "text-valid")} tabIndex='0'>
+          {message.length} / 140
+        </p>
+        <button
+          className='submit-button'
+          type='submit'
+          onClick={handleSubmit}
+          disabled={message.length < 5 || message.length > 140 ? true : false}
```

Comment:
This is merely a personal reflection, but I've come across a lot of discussions on using disabled buttons from an accessibility point of perspective. Here's a short text on it, if you're interested: 
[Is it ok to ‘grey out’ disabled buttons?](https://uxdesign.cc/is-it-ok-to-grey-out-disabled-buttons-8afa74a0fae)

Reply:
Thank you, will check this out